lib/sorbet-runtime-stub.rb

```diff
@@ -1,5 +1,11 @@
 # typed: ignore
 
+begin
+  gem "sorbet-runtime"
```

Comment:
Is working with bundler? Will it load `sorbet-runtime` from my system if it exists?

Reply:
Indeed it is working with `bundler` and, no, it won't load a random gem file that is not listed in your Gemfile:

```
$ pry
[1] pry(main)> gem "diffy"
=> true

$ bundle exec pry
[1] pry(main)> gem "diffy"
Gem::LoadError: diffy is not part of the bundle. Add it to your Gemfile.
from /Users/ufuk/.gem/ruby/2.6.5/gems/bundler-1.17.3/lib/bundler/rubygems_integration.rb:408:in `block (2 levels) in replace_gem'
```

Since `bundler` hooks the `Kernel#gem` method to ensure that it only loads the gems that are in the Gemfile.

Reply:
Cool!